pyro/contrib/zuko.py

```diff
@@ -0,0 +1,46 @@
+# Copyright Contributors to the Pyro project.
+# SPDX-License-Identifier: Apache-2.0
+
+import torch
+from torch import Size, Tensor
+
+import pyro
+
+
+class Zuko2Pyro(pyro.distributions.TorchDistribution):
+    r"""Wraps a Zuko (or PyTorch) distribution as a Pyro distribution."""
```

Comment:
can you expand this docstring? link to zuko? etc

also the `(or PyTorch)` seems confusing since we have `pyro.distributions.TorchDistribution` for that

Reply:
I don't think `TorchDistribution` does the same think. `Zuko2Pyro` takes any **instance** of distribution and makes it a valid Pyro distribution.

I'll add a page in the docs.